### PR TITLE
Add issue to Fleet troubleshooting page for ECE 2.10 / Fleet 7.14 issue

### PR DESCRIPTION
**Pull request by kilfoyle**
This adds an entry to the 7.14 Fleet [troubleshooting page](https://www.elastic.co/guide/en/fleet/7.14/fleet-troubleshooting.html), since version 7.14 of Fleet is not working with the recently released ECE 2.10. Here's an [issue with the details](https://github.com/elastic/support-known-issues/issues/791). I'm putting similar warnings in the Cloud ECE docs

Please note that the workaround isn't ready yet so we need to wait for that before merging.

I'd really appreciate any help with labels, and I assume there's no backporting since this affects and targets only the 7.14 release.

### CODE REVIEW

**Review by AlexP-Elastic — COMMENTED** [review]
LGTM (but I'm waiting on a few things before re-issuing the stackpack, prob tomorrow, so hold off on merging)

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Review by bmorelli25 — APPROVED** [review]
Thanks, @kilfoyle!

File: docs/en/ingest-management/troubleshooting.asciidoc
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:438]:
```suggestion
When an {ece} deployment is set up with APM and {fleet}, even though the deployment operates as expected, {fleet} cannot be configured. Agent log files may contain a message like the following:
```

File: docs/en/ingest-management/troubleshooting.asciidoc
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:445]:
```suggestion
This is a known issue with the {stack} pack version 7.14 shipped with {ece} version 2.10. To resolve the problem, link:{ece-ref}/ece-manage-elastic-stack.html[re-upload a fresh copy] of the version 7.14 {stack} pack to overwrite the original one. This issue will also be addressed in later {stack} packs and {ece} versions.
```

**Review by dedemorton — COMMENTED** [review]
Thanks for doing this! Just a few minor suggestions.

File: docs/en/ingest-management/troubleshooting.asciidoc
Comment (dedemorton) [reply docs/en/ingest-management/troubleshooting.asciidoc:445]: I'm assuming here that "pack" is an ECE-related term because we don't use "pack" in the stack docs (AFAIK).

Comment (kilfoyle) [reply docs/en/ingest-management/troubleshooting.asciidoc:445]: Yup, that's exactly right. ECE comes with "stack pack" archived bundles of the stack products.

**Review by dedemorton — APPROVED** [review]
LGTM!

File: docs/en/ingest-management/troubleshooting.asciidoc
Comment (kilfoyle) [inline docs/en/ingest-management/troubleshooting.asciidoc:445]:
```suggestion
This is a known issue with the {stack} pack version 7.14 and {ece} version 2.10. With the original {stack} pack version 7.14, if downloaded from the Elastic website before August 10, 2021, Fleet does not work when enabled in a deployment. 

To resolve the problem, link:{ece-ref}/ece-manage-elastic-stack.html[get and re-upload a fresh copy] of the version 7.14 Elastic {stack} pack to overwrite the original one. If you have existing version 7.14 deployments, then restart Fleet/APM after re-uploading the Elastic {stack} pack to enable Fleet. This issue will be addressed in later {stack} packs and {s} versions.
```

**Comment by kilfoyle** [comment]
Thanks Brandon and DeDe for the quick review! I've updated the text as suggested and also based on Alex's suggestions in the equivalent Cloud docs PR.

I'll wait until the revised stack pack is ready before adjusting the "if downloaded before" date if needed and ii) merging.

**Comment by AlexP-Elastic** [comment]
@kilfoyle thanks for all the work on this! The stackpack is updated now, so you are good to merge these PRs whenever you're ready <3

**Pull request merged**
